mapit_gb/management/commands/mapit_UK_add_slugs_to_local_authorities.py

```diff
@@ -0,0 +1,38 @@
+# This script is for a one off import of govuk slugs for local authorities
+
+import json
+from django.core.management.base import NoArgsCommand
+from mapit.models import Area, CodeType
+from django.utils.six.moves import urllib
+
+
+class Command(NoArgsCommand):
+    help = 'Assigns slugs from frontend to local authorities'
+
+    def handle_noargs(self, **options):
+        try:
+            code_type = CodeType.objects.get(code='govuk_slug')
+        except CodeType.DoesNotExist:
+            code_type = CodeType(
+                code='govuk_slug',
+                description='Slug for use by GOV.UK'
+            )
+            code_type.save()
+
+        url = "https://raw.githubusercontent.com/alphagov/frontend/master/lib/data/authorities.json"
```

Comment:
I think skipping straight to step 3 sounds reasonable. I've added the data file and the importer uses it to get the slugs.